src/main/java/com/cloudogu/scm/review/pullrequest/api/EventSubscriptionRequest.java

```diff
@@ -18,6 +19,15 @@
   @PathParam("pullRequestId")
   private String pullRequestId;
 
-  @HeaderParam("X-SCM-Session-ID")
-  private SessionId sessionId;
+  @HeaderParam(SessionId.PARAMETER)
```

Comment:
We should not use the constant from core, because it is not available to version prior to the upcoming RC-6 of SCM-Manager.

Reply:
This is not used in bugfix commit for prior releases.